ra/ra_test.go

```diff
@@ -1869,7 +1869,9 @@ func TestDeactivateAuthorization(t *testing.T) {
 	exp := ra.clk.Now().Add(365 * 24 * time.Hour)
 	authzID := createFinalizedAuthorization(t, sa, "not-example.com", exp, "valid")
 	authz := getAuthorization(t, fmt.Sprintf("%d", authzID), sa)
-	err := ra.DeactivateAuthorization(ctx, authz)
+	pbAuthz, err := bgrpc.AuthzToPB(authz)
```

Comment:
Feels very odd that the `getAuthorization` helper which provides this `authz` object
a) does so by converting a PB to a Core object immediately before returning; and
b) has its result converted back into a PB immediately at 60% of its call-sites
Better to change its return value to be the protobuf format; or maybe both formats.

Reply:
Awesome! I've managed to just have it return a corepb.Authorization instead